lib/src/view/explorer/explorer_view.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:dartchess/dartchess.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:lichess_mobile/src/model/common/chess.dart';
+import 'package:lichess_mobile/src/utils/l10n_context.dart';
+import 'package:lichess_mobile/src/view/explorer/opening_explorer_view.dart';
+import 'package:lichess_mobile/src/view/explorer/tablebase_view.dart';
+
+/// Unified explorer view that shows either opening explorer or tablebase
+/// based on the position state (opening vs endgame)
+
+const kTableRowVerticalPadding = 10.0;
+const kTableRowHorizontalPadding = 8.0;
+const kTableRowPadding = EdgeInsets.symmetric(
+  horizontal: kTableRowHorizontalPadding,
+  vertical: kTableRowVerticalPadding,
+);
+const kHeaderTextStyle = TextStyle(fontSize: 12);
+
+Color whiteBoxColor(BuildContext context) => Theme.of(context).brightness == Brightness.dark
+    ? Colors.white.withValues(alpha: 0.8)
+    : Colors.white;
+
+Color blackBoxColor(BuildContext context) => Theme.of(context).brightness == Brightness.light
+    ? Colors.black.withValues(alpha: 0.7)
+    : Colors.black;
+
+class ExplorerView extends ConsumerWidget {
+  const ExplorerView({
+    required this.position,
+    required this.onMoveSelected,
+    this.opening,
+    this.isComputerAnalysisAllowed = true,
```

Comment:
Perhaps it was already a default in the previous implementation, but I'd rather not have one here, as it is better to be sure that computer analysis is not allowed by mistake.